src/lib/dnssd/Advertiser.h

```diff
@@ -101,6 +101,13 @@ class BaseAdvertisingParams
     }
     Optional<bool> GetTcpSupported() const { return mTcpSupported; }
 
+    Derived & SetICDOperatesAsLIT(Optional<bool> ICDOperatesAsLIT)
```

Comment:
```suggestion
    Derived & SetICDOperatesAsLIT(Optional<bool> operatesAsLIT)
```
to match sfile style... or `icdOperatesAsLIT`, I guess.

Reply:
Didn't we discuss before that ICD was an acronym and therefore should be always capital? 
I will use `operatesAsLIT` when it is clear enough that this is tied to ICD like here. Otherwise I'll move the ICD later in the variable name

Reply:
It's an acronym, so should not be "Icd".  It should be "ICD" or "icd"; but the point is it gets capitalized or not as a unit...

Reply:
Should always be uppercase in my opinion. Anyhow, I renamed the vars  in a way that addresses this comment